.gitignore

```diff
@@ -211,3 +211,8 @@ project.lock.json
 #####
 
 logs/
+
+tools/AndroidDependencies/*
```

Comment:
Why are we going for an "ignore all by default and whitelist" approach rather than the other way around? Also it'd be better for there to be a gitignore within the AndroidDependencies directory instead

Reply:
Can move that there.